OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -187,8 +161,15 @@ public override Activity Tick(Actor self)
 
 			// If the actor is inside a tunnel then we must let them move
 			// all the way through before moving to the next activity
-			if (IsCanceled && self.Location.Layer != CustomMovementLayerType.Tunnel)
+			if (IsCanceling && self.Location.Layer != CustomMovementLayerType.Tunnel)
+			{
+				// Although MoveFirstHalf and MoveSecondHalf can't be interrupted,
```

Comment:
This change looks wrong. If we get to this point then we're guaranteed to move on to the next activity, so we never access `path` again.  Either clearing this is redundant (and can be removed), or this needs to move somewhere else.

Reply:
Yes, clearing the path is redundant now; Removed.